data.json

```
@@ -1324,5 +1324,11 @@
     "githubUsername": "Krishnapro",
     "imageUrl": "https://user-images.githubusercontent.com/59277887/132136661-d62a6664-0fa3-437d-b97d-ea41ca21c61f.png",
     "issueNumber": 702
+  },
+  {
+  "name": "Omkar Kulkarni",
+  "githubUsername": "Omkar0114",
+  "imageUrl": "https://user-images.githubusercontent.com/88308267/132665808-d55d32e9-a79c-4c7f-8932-0f20c9442a6f.png",
+  "issueNumber": 710
```

also please take a look at the spacing

```suggestion
    "name": "Omkar Kulkarni",
    "githubUsername": "Omkar0114",
    "imageUrl": "https://user-images.githubusercontent.com/88308267/132665808-d55d32e9-a79c-4c7f-8932-0f20c9442a6f.png",
    "issueNumber": 710
```